CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 - [operation] Rework `OperationHandler` to provide an optional command instead of direct execution to allow more execution control [#187](https://github.com/eclipse-glsp/glsp-server/pull/187)
   - `Abstract<XYZ>` base implementations were deprecated in favor of `GModelOperationHandler` and `EMFOperationHandler` base classes
   - Long-term deprecated and unused `Basic<XYZ>` base classes were removed
+- [server] Default port has changed from 5007 to 0, which implies autoassignment by the OS
```

Comment:
It don`t think it's necessary to split this into bullet points. One pullet points covering both should be enough. Please also adda link to the PR at the end like this: [#198](https://github.com/eclipse-glsp/glsp-server/pull/198/)